google/gax/auth.py

```diff
@@ -31,19 +31,49 @@
 
 from __future__ import absolute_import
 
+import grpc
```

Comment:
Actually - I think I'm going to shove this into `_grpc_oauth2client.py`, so that I can add `_grpc_google_auth.py` with the same interface.